spec/docs/v3/statements_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 require "swagger_helper"
 
-RSpec.describe "API", type: :request, swagger_doc: "v3/api_spec.json" do
+RSpec.describe "API", type: :request, swagger_doc: "v3/api_spec.json", api_v3: true do
```

Comment:
we've already added some of those endpoints. Will fix/enable in another PR as specs need updating